business/sdk/sqldb/sqldb.go

```diff
@@ -10,12 +10,13 @@ import (
 	"strings"
 	"time"
 
-	"github.com/ardanlabs/service/foundation/logger"
-	"github.com/ardanlabs/service/foundation/tracer"
 	"github.com/jackc/pgx/v5/pgconn"
 	_ "github.com/jackc/pgx/v5/stdlib"
 	"github.com/jmoiron/sqlx"
 	"go.opentelemetry.io/otel/attribute"
+
```

Comment:
I like the change except the imports. Just keep 2 sets. Stdlib and non-stdlib

if you are not in the CONTRIBUTORS file can you add yourself please